pkgs/top-level/aliases.nix

```
@@ -1793,6 +1793,7 @@ mapAliases ({
   tomcat8 = throw "tomcat8 has been removed from nixpkgs as it has reached end of life"; # Added 2021-06-16
   tomcat85 = throw "tomcat85 has been removed from nixpkgs as it has reached end of life"; # Added 2020-03-11
   tor-arm = throw "tor-arm has been removed from nixpkgs as the upstream project has been abandoned"; # Added 2022-01-01
+  tor-browser-bundle-bin = tor-browser; # Added 2023-09-23
   torbrowser = throw "'torbrowser' has been renamed to/replaced by 'tor-browser-bundle-bin'"; # Converted to throw 2022-02-22
```

```suggestion
  torbrowser = throw "'torbrowser' has been renamed to/replaced by 'tor-browser'"; # Converted to throw 2022-02-22
```

Similarly further down in this file for `tor-browser-bundle` and `tor-browser-unwrapped`, let the error message hint to use `tor-browser` instead of `tor-browser-bundle-bin`.

---

Good point.